### PR TITLE
Perf Tests: Use in memory storage for scheduler

### PR DESCRIPTION
**Pull request by JoshVanL**
# Description

## Issue reference

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Review by artursouza — DISMISSED** [review]
Pending test results.

**Review by cicoyle — APPROVED** [review]

**Review by artursouza — DISMISSED** [review]

File: tests/perf/actor_reminder/actor_reminder_test.go [hunk 1]
```diff
@@ -352,6 +378,27 @@ func TestActorReminderTriggerPerformance(t *testing.T) {
...
+		var pod corev1.Pod
+		err = cl.Get(context.Background(), client.ObjectKey{Namespace: kube.DaprTestNamespace, Name: "dapr-scheduler-server-0"}, &pod)
+		require.NoError(t, err)
+		err = cl.Delete(context.Background(), &pod)
+		require.NoError(t, err)
+
+		assert.EventuallyWithT(t, func(c *assert.CollectT) {
+			var sts appsv1.StatefulSet
+			err = cl.Get(context.Background(), client.ObjectKey{Namespace: kube.DaprTestNamespace, Name: "dapr-scheduler-server"}, &sts)
+			require.NoError(t, err)
```
Comment (cicoyle) [inline tests/perf/actor_reminder/actor_reminder_test.go:397]:
```suggestion
			require.NoError(c, err)
```

**Review by artursouza — APPROVED** [review]

**Pull request merged**